addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -182,23 +182,23 @@ pscanrules.crossdomainscriptinclusion.soln=Ensure JavaScript source files are lo
 
 pscanrules.csp.name=CSP
 pscanrules.csp.desc=Content Security Policy (CSP) is an added layer of security that helps to detect and mitigate certain types of attacks. Including (but not limited to) Cross Site Scripting (XSS), and data injection attacks. These attacks are used for everything from data theft to site defacement or distribution of malware. CSP provides a set of standard HTTP headers that allow website owners to declare approved sources of content that browsers should be allowed to load on that page \u2014 covered types are JavaScript, CSS, HTML frames, fonts, images and embeddable objects such as Java applets, ActiveX, audio and video files.
-pscanrules.csp.desc.extended=\n\nThe directive(s): {0} are among the directives that do not fallback to default-src, missing/excluding them is the same as allowing anything.
+pscanrules.csp.otherinfo.extended=\n\nThe directive(s): {0} are among the directives that do not fallback to default-src, missing/excluding them is the same as allowing anything.
```

Comment:
Is this still extending another string? Are the leading new lines still correct?

Reply:
Yes

Reply:
Gets extended in the otherinfo field now